### PR TITLE
Kills #C8A5DC, adds syndie pill color, nerfs xenobiology

### PR DESCRIPTION
**Pull request by carlarctg**
## About The Pull Request
Title. Eliminates every single instance of the default violet color, minus one that gets overwritten in init. Also adds a syndie pill, red and black, and makes regenerative jelly dye your hair purple.

## Why It's Good For The Game
I hate these lazy default colors they suck and ruin the game IT'S A 2 SECOND LINE CHANGE ITS NOT THAT HARD OKAY WHY MAKE YOUR CHEMICAL SO BORING

Xenobiology is just way too strong, it's not balanced. To counter this, Regenerative Jelly will now dye your hair purple to incentivize xenobiologists to leave their nest and utilize COBBCHEM instead.

## Changelog
:cl:
imageadd: KILLED default chemical color
imageadd: added syndie pill, completely unused as of now please PR stuff with it because it's cool
tweak: regen jelly makes hair purple
/:cl:

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
Fix your line endings you have 3 files completely changed

**Comment by carlarctg** [comment]
wow i fucked a lot of stuff up

**Comment by HomocidlePancakes** [comment]
I like C8A5DC

**Comment by Angustmeta** [comment]
Hell it's about time

**Comment by carlarctg** [comment]
> I like C8A5DC

I don't like you

**Comment by fludd12** [comment]
PURPLE HAIR GANG RISE UP

**Comment by HomocidlePancakes** [comment]
>I don't like you

Neither do I. You’ve tried to stealthily remove Reimus outfit in your reflector PR. I represent the 2hu metagang and I have a message for you - cease or get buttfucked.

**Comment by ATH1909** [comment]
default violet?
more like
default dance

i'm sorry
it's 1:30 AM and I'm tired and procrastinating

**Comment by Couls** [comment]
who measures line changes in seconds? 🚲🏠

syndie pill looks cool tho

**Comment by Repukan** [comment]
Make regen jelly make you bald

Sting where it hurts

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 1]
```diff
@@ -1473,43 +1473,45 @@
 	taste_description = "blueyalty" //also intentional
 	carpet_type = /turf/open/floor/carpet/royalblue
 
-/datum/reagent/bromine
+/datum/reagent/bromine //bromine moment
 	name = "Bromine"
 	description = "A brownish liquid that's highly reactive. Useful for stopping free radicals, but not intended for human consumption."
 	reagent_state = LIQUID
-	color = "#C8A5DC"
+	color = "D35415"
```
Comment (MrPerson) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: You dropped the pound sign here

File: code/modules/reagents/chemistry/reagents/other_reagents.dm [hunk 2]
```diff
@@ -1535,10 +1537,13 @@
 	name = "Quantum Hair Dye"
 	description = "Has a high chance of making you look like a mad scientist."
 	reagent_state = LIQUID
-	color = "#C8A5DC"
+	color = "#C8A5DC" //gets overwritten
 	var/list/potential_colors = list("0ad","a0f","f73","d14","d14","0b5","0ad","f73","fc2","084","05e","d22","fa0") // fucking hair code
 	taste_description = "sourness"
 
+/datum/reagent/colorful_reagent/Initialize()
+	color = pick(random_color_list)
```
Comment (MrPerson) [inline code/modules/reagents/chemistry/reagents/other_reagents.dm]: I think this is supposed to be on hair_dye, not colorful_reagent

**Comment by AnonymousNow** [comment]
SS13XENO

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 3]
```diff
@@ -861,16 +860,21 @@
...
 	M.adjustBruteLoss(-1.5*REM, 0)
 	M.adjustFireLoss(-1.5*REM, 0)
 	M.adjustOxyLoss(-1.5*REM, 0)
 	M.adjustToxLoss(-1.5*REM, 0, TRUE) //heals TOXINLOVERs
-	. = 1
+	if(ishuman(M)
+	var/mob/living/carbon/human/H = M
+		M.hair_color = "92f"
+		M.facial_hair_color = "92f"
+		M.update_hair()
```
Comment (nemvar) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: :thinking:

**Comment by Grazyn** [comment]
So now stabilized rainbow with regenerative extract will automatically dye your hair purple when you enter crit. Cool!

As if xenobio didn't have enough pointless extracts that are never used.

**Comment by Ty-the-Smonk** [comment]
Cobby said regen jelly was fine, but if you are changing regen extracts, then make it something less shit please

**Comment by carlarctg** [comment]
nowhere in the pr does it say i'm removing jelly's healing effect

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm (hunk 3 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: don't worry it doesn't work

**Comment by Krysonism** [comment]
There is already a medicine that colours hair purple.

`/datum/reagent/medicine/polypyr/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
	if(method == TOUCH || method == VAPOR)
		if(M && ishuman(M) && reac_volume >= 0.5)
			var/mob/living/carbon/human/H = M
			H.hair_color = "92f"
			H.facial_hair_color = "92f"
			H.update_hair()`

**Comment by carlarctg** [comment]
> There is already a medicine that colours hair purple.
> 
> ```
> /datum/reagent/medicine/polypyr/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
> 	if(method == TOUCH || method == VAPOR)
> 		if(M && ishuman(M) && reac_volume >= 0.5)
> 			var/mob/living/carbon/human/H = M
> 			H.hair_color = "92f"
> 			H.facial_hair_color = "92f"
> 			H.update_hair()
> ```

exactly

**Comment by spessbandit** [comment]
what did #C8A5DC ever do to you

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm [hunk 4]
```diff
@@ -629,11 +619,20 @@
...
 
+/datum/reagent/medicine/inacusiate
+	name = "Inacusiate"
+	description = "Instantly restores all hearing to the patient, but does not cure deafness."
+	color = "#606060" // ditto
+
+/datum/reagent/medicine/inacusiate/on_mob_life(mob/living/carbon/M)
+	M.restoreEars()
+	..()
+
```
Comment (Iamgoofball) [inline code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: just adding a chemical I guess

File: code/modules/reagents/chemistry/reagents/other_reagents.dm (hunk 2 above)
Comment (Iamgoofball) [reply code/modules/reagents/chemistry/reagents/other_reagents.dm]: Nah, it's randomized.

File: code/modules/reagents/chemistry/recipes/medicine.dm [hunk 5]
```diff
@@ -18,11 +18,19 @@
...
 	required_reagents = list(/datum/reagent/cryptobiolin = 1, /datum/reagent/medicine/epinephrine = 1)
 
+/datum/chemical_reaction/oculine
+	name = "Oculine"
+	id = /datum/reagent/medicine/oculine
+	results = list(/datum/reagent/medicine/oculine = 3)
+	required_reagents = list(/datum/reagent/medicine/C2/multiver = 1, /datum/reagent/carbon = 1, /datum/reagent/hydrogen = 1)
+	mix_message = "The mixture bubbles noticeably and becomes a dark grey color!"
+
 /datum/chemical_reaction/inacusiate
```
Comment (Iamgoofball) [inline code/modules/reagents/chemistry/recipes/medicine.dm]: You need to clean your branch.

File: code/modules/reagents/chemistry/reagents/medicine_reagents.dm (hunk 4 above)
Comment (carlarctg) [reply code/modules/reagents/chemistry/reagents/medicine_reagents.dm]: i moved inacusiate next to oculine

File: code/modules/reagents/chemistry/recipes/medicine.dm (hunk 5 above)
Comment (carlarctg) [reply code/modules/reagents/chemistry/recipes/medicine.dm]: same, moved oculine next to inacusiate

**Review by actioninja — CHANGES_REQUESTED** [review]
Clean up the branch, you've got fucked line ending files and random junk all over.

**Comment by optimumtact** [comment]
stale

**Pull request closed without merging**